### PR TITLE
GitHub Website update

### PR DESCRIPTION
**Pull request by Aericio**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

updating myplot site on github

## Optional information
i was bored, just reformatting, deleting/cleaning up some things.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -1,120 +1,112 @@
...
-					jQuery('.back-to-top').click(function (event) {
-						event.preventDefault();
-						jQuery('html, body').animate({scrollTop: 0}, duration);
-						return false;
-					})
-				});
-			</script>
-		</div>
-	</body>
+<head>
```
Comment (jasonwynn10) [inline about.html]: indents should be tabs instead of spaces

File: contributors.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
 		<meta charset='UTF-8'>
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-		<meta name="description" content="MyPlot : An MCPE Plot Management Plugin">
+		<meta name="description" content="MyPlot: MCPE Plot Management Plugin">
```
Comment (jasonwynn10) [inline contributors.html]: leave in the "An"

File: contributors.html [hunk 3]
```diff
@@ -73,7 +72,7 @@ <h3 id="project_tagline">Welcome to the official MyPlot wiki</h3>
 				<div class="contributors">
 					<img src="https://avatars0.githubusercontent.com/u/16523741?v=3&s=460">
 					<h2><a href="https://github.com/Aericio">Aericio</a></h2>
-					<p>Aerico joined the MyPlot project to help with testing.</p>
+					<p>Aerico joined the MyPlot project to help with testing and little bug fixes.</p>
```
Comment (jasonwynn10) [inline contributors.html]: "a few" would fit better than "a little"

File: downloads.html [hunk 4]
```diff
@@ -5,7 +5,7 @@
 		<meta charset='UTF-8'>
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
-		<meta name="description" content="MyPlot : An MCPE Plot Management Plugin">
+		<meta name="description" content="MyPlot: MCPE Plot Management Plugin">
```
Comment (jasonwynn10) [inline downloads.html]: again, leave in the "An"

File: about.html (hunk 1 above)
Comment (Aericio) [inline about.html]: phpstorm

Comment (Aericio) [inline about.html]: they are tabs.

Comment (jasonwynn10) [inline about.html]: line 4 and below uses spaces

Comment (Aericio) [inline about.html]: i see, opened with npp and they were spaces. i don't feel like fixing the whole thing to tabs... so i'm just going to close this.

**Comment by Aericio** [comment]
https://stackoverflow.com/questions/5430806/html-indenting-standard-tab-or-two-spaces people say it doesn't matter, besides it saves a couple bytes. phpstorm uses html5 boilerplate which uses two spaces.

> As nobody mentioned it, the Google HTML/CSS Style Guide and the W3School HTML(5) Style Guide recommend 2 spaces.

**Comment by jasonwynn10** [comment]
Whatever works, I would prefer consistency though

**Comment by Aericio** [comment]
you're choice if you want to merge or not, but when reformatted with phpstorm that's what it did ¯\_(ツ)_/¯

**Comment by jasonwynn10** [comment]
I'll make some consistency changes, then merge it

**Comment by Aericio** [comment]
👌

**Pull request merged**